packages/mdc-button/_mixins.scss

```diff
@@ -60,6 +60,10 @@
   border-radius: $corner-radius;
 }
 
+@mixin mdc-button-horizontal-padding($left, $right: $left) {
```

Comment:
This mixin should only need to be passed a single value that will be used for both margins.

The current implementation optimizes for certain specific icons, but causes other icons to become off-center in relation to the button's padding (such as both icons used in the demo page!). Icons are assumed to occupy a 18x18 space; whether the actual stroke/fill of the icon does is beyond our control.

If we simplify this, we also should not need the new `--with-icon` modifier class at all, which means we don't need breaking changes to the component.

Reply:
I got the feeling that this cannot be simplified any further.